components/markets/ScalarPriceRange.tsx

```diff
@@ -6,7 +6,7 @@ import type { ScalarRangeType } from "@zeitgeistpm/sdk/dist/types";
 import moment from "moment";
 
 interface ScalarPriceRangeProps {
-  scalarType: ScalarRangeType;
+  scalarType: string | ScalarRangeType;
```

Comment:
What's the reason for the union type here? I thought `ScalarRangeType` should cover it

Reply:
I thought so too but when I traced it back it seems to be coming from the sdk which has assigned it a type of `string | null`. I can talk to @yornaath and see what might be the issue

Reply:
I needed to assert `scalarType` to `const ` and that sorted it out